lib/QuadTermUtil.ts

```diff
@@ -307,22 +307,32 @@ export function matchPatternComplete(quad: RDF.BaseQuad, pattern: RDF.BaseQuad):
  *
  * @param quad A quad - possibly containing variables
  * @param pattern A pattern - possibly containing variables
+ * @param options
+ *    skipVar - don't add variables in the quad to the mapping
```

Comment:
What does this mean exactly?
Will variables in the quad match with anything from the pattern?

Reply:
Let quad be `?s a ?s` and pattern be `?a ?b ?c`.

If `skipVar` is set to true, then this match is fine. If `skipVar` is set to false then there is a conflict between trying to set ?s = ?a and ?s = ?c so it returns false

Reply:
Ok, that's clear. We'll need a more descriptive comment to explain this behaviour though (and maybe also a different option name, but can't think of any suggestions atm).

Reply:
Perhaps mapVar, where mapVar = !skipVar?

Reply:
Not sure. Maybe something in the direction of `skipVarIdentity`?

Reply:
Not sure - `skipVarMapping`?

Reply:
or `skipMappingVar`?